internal/config/config.go

```diff
@@ -21,9 +21,10 @@ import (
 var (
 	Config = &configuration{
 		Server: server{
-			Address: "127.0.0.1",
-			Port:    7200,
-			Token:   "",
+			Address:                 "127.0.0.1",
+			Port:                    7200,
+			SystemdSocketActivation: true,
```

Comment:
To be honest, I am not sure about that default. Let me explain why:

If I read the code correctly, we assume the Systemd socket by default. If that socket is, however, not configured, Poseidon won't start. Now, consider an upgrade: Someone is _just_ upgrading to the latest release, without changing the systemd files or the config: Poseidon will break. Therefore, I would vote to either have that new option as an opt-in (default false) or alternatively have a fallback to the traditional, non-systemd usage (default true, try socket, if no socket available, start with given address and port). Either way, upgrading would work without an error.